app/Battle/great_battle.py

```diff
@@ -0,0 +1,14 @@
+from app.Knights.knight_class import Knight
+
+
+def battle_action_phase(knight_1: Knight, knight_2: Knight) -> None:
```

Comment:
You should use verbs for function naming to describe an action that is performed by a function. Rename it to be more descriptive. You could use `perform_fight` here as an example.

Reply:
👍